canonical-websites/www.ubuntu.com
# /support/esm

## Done

Created the new ESM page in the support section and fixed the news navigation for top-level nav items.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/support/esm](http://0.0.0.0:8001/support/esm))
- Check the page in all viewports
- Check the [copy](https://docs.google.com/document/d/1BK-XXECYJJllG8jIe3WN3X8EMo_3wBBa03nlzYDP8xE/edit#)
- Ensure that the navigation and breadcrumbs look correct at all viewports.

[Demo](http://www.ubuntu.com-vbt-support-master.demo.haus/support/esm)
